src/models.jl

```diff
@@ -62,3 +67,13 @@ struct PrimitiveEquationModel{NF<:AbstractFloat,D<:AbstractDevice} <: ModelSetup
     implicit::Implicit{NF}
     device_setup::DeviceSetup{D}
 end
+
+has(::Type{PrimitiveEquationModel{NF,D}}, var_name::Symbol) where {NF,D} = var_name in [:vor, :div, :temp, :humid, :pres]
+
+"""
+    has(::ModelSetup, var_name::Symbol)
+
+Returns true if the model `M` has a prognostic variable `var_name`, false otherwise. The default fallback is that all variables are included. 
+"""
+has(::Type{ModelSetup{D}}, var_name::Symbol) where D = var_name in [:vor, :div, :temp, :humid, :pres] 
```

Comment:
It should also be possible to drop it here? Or do you see any advantage of keeping it?